docs/installing_deis/gce.rst

```diff
@@ -95,47 +95,41 @@ Create a SSH key that we will use for Deis host communication:
-
-    Table of resources:
-
-    +--------+---------------+--------+---------+
-    | name   | zone          | status | size-gb |
-    +--------+---------------+--------+---------+
-    | cored1 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored2 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored3 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
+    $ gcloud compute disks create cored1 cored2 cored3 --size 64GB --type pd-ssd --zone us-central1-f
 
+    NAME   ZONE          SIZE_GB TYPE   STATUS
+    cored1 us-central1-f 64      pd-ssd READY
+    cored2 us-central1-f 64      pd-ssd READY
+    cored3 us-central1-f 64      pd-ssd READY
 
 Launch 3 instances. You can choose another starting CoreOS image from the listing output of
 ``gcloud compute images list``:
 
 .. code-block:: console
 
-    $ for num in 1 2 3; do gcutil addinstance --use_compute_key --image projects/coreos-cloud/global/images/coreos-stable-557-2-0-v20150210 --persistent_boot_disk --zone us-central1-a --machine_type n1-standard-2 --tags deis --metadata_from_file user-data:gce-user-data --disk cored${num},deviceName=coredocker --authorized_ssh_keys=core:~/.ssh/deis.pub,core:~/.ssh/google_compute_engine.pub core${num}; done
-
-    Table of resources:
-
-    +-------+---------------+--------------+---------------+---------+
-    | name  | network-ip    | external-ip  | zone          | status  |
-    +-------+---------------+--------------+---------------+---------+
-    | core1 | 10.240.33.107 | 23.236.59.66 | us-central1-a | RUNNING |
-    +-------+---------------+--------------+---------------+---------+
-    | core2 | 10.240.94.33  | 108.59.80.17 | us-central1-a | RUNNING |
-    +-------+---------------+--------------+---------------+---------+
-    | core3 | 10.240.28.163 | 108.59.85.85 | us-central1-a | RUNNING |
-    +-------+---------------+--------------+---------------+---------+
+    $ for num in 1 2 3; do
+      gcloud compute instances create core${num} \
+      --zone us-central1-f \
+      --machine-type n1-highmem-2 \
+      --metadata-from-file user-data=gce-user-data sshKeys=~/.ssh/deis.pub \
+      --disk name=cored${num} device-name=coredocker \
+      --tags deis \
+      --image coreos-stable-557-2-0-v20150210 \
+      --image-project coreos-cloud;
+    done
+
+    NAME  ZONE          MACHINE_TYPE INTERNAL_IP   EXTERNAL_IP    STATUS
+    core1 us-central1-f n1-highmem-2 10.240.10.107 108.59.80.10   RUNNING
+    core2 us-central1-f n1-highmem-2 10.240.10.108 108.59.80.11   RUNNING
+    core3 us-central1-f n1-highmem-2 10.240.10.109 108.59.80.12   RUNNING
 
 .. note::
 
-    The provision script will by default provision ``n1-standard-2`` instances. Choosing a smaller
+    The provision script will by default provision ``n1-highmem-2`` instances. Choosing a smaller
```

Comment:
What's the motivation for this change? Is `standard` not large enough? If we want to make this change, it should be at the minimum a separate commit (or even a separate PR).